tools/urtrace/urtrace

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: MIT
+import argparse
+import subprocess
+import os
+import sys
+
+def find_library(paths, name, recursive=False):
+    for path in paths:
+        for root, _, files in os.walk(path):
+            for f in files:
+                if f == name:
+                    return os.path.abspath(os.path.join(root, name))
+            if not recursive:
+                break
```

Comment:
This could be replaced with a simple:

glob.glob(f'**/{name}', root_dir=path, recursive=recursive)

Reply:
I'd argue if it's simpler (it's a couple characters less, I'll give you that :P), but done.